### DISCUSSION TITLE
how do i use assetripper on mac?

### DISCUSSION THREAD

**Discussion by oo1o1o12** [AssetRipper/AssetRipper#1309] [Q&A]
what do i do to launch the gui? there is no executable in the folder anymore

**Comment by ds5678** [comment]
Did you download source code? That would explain why you don't have an executable.

**Reply by oo1o1o12** [reply]
no, i downloaded the x64 zip file, the older versions have an executable

**Reply by oo1o1o12** [reply]
there is only some "gui.free" file but there is NO WAY TO OPEN IT

**Comment by mextex** [comment]
I also don't know how to launch assetripper on my mac. I downloaded the latest release package "AssetRipper_mac_x64.zip" and have a file called "AssetRipper.GUI.Free" in there. But I can't open it.

**Comment by mextex** [comment]
Thank you. And could you give me a hint, how launch AssetRipper on a mac?

**Comment by ntzavaras** [comment]
@mextex take a look here: https://assetripper.github.io/AssetRipper/articles/RunningOnMac.html
